lib/server.js

```diff
@@ -201,6 +216,11 @@ function main() {
     if (process.env.S3BACKEND === 'mem') {
         clusters = 1;
     }
+    if (clusters === 1) {
+        const server = new S3Server();
+        server.initiateStartup(logger.newRequestLogger());
+        return;
```

Comment:
You will skip the call to `monitoringClient.collectDefaultMetrics({ timeout: 5000 });` by returning immediately, is this intended?